dockerfiles/Dockerfile.browser

```diff
@@ -0,0 +1,34 @@
+# Copyright Radiant Earth Foundation
```

Comment:
this is almost a copy of their dockerfile, with the exception of the git cloning. This slight change is necessary so that I can overwrite `config.js` with the custom one I added in our repo.